CONTRIBUTING.md

```diff
@@ -78,6 +78,21 @@ The following GitHub users are the currently available sponsors, either by being
 * [@djaglowski](https://github.com/djaglowski)
 
 Whenever a sponsor is picked from the top of this list, please move them to the bottom.
+
+## Adding New Metrics
+Following these steps for contributing additional metrics.
+ - Make sure you have basic tools, like `golang, make and git` installed in your system. 
+ - Fork this repo and check it out locally to your system, e.g. into the directory `opentelemetry-collector-contrib`
+ - Make sure $GOPATH is added to $PATH, if not sure, run `export PATH=$PATH:$(go env GOPATH)/bin`
+ - Run `cd opentelemetry-collector-contrib`
+ - Run `make install-tools`
+ - Edit `metadata.yaml` of your metrics receiver to add new metrics, e.g.: `redisreceiver/metadata.yaml`
+ - To generate new metrics on top of this updated YAML file.
+   - Run `cd receiver/redisreceiver`
+   - Run `go generate`
```

Comment:
```suggestion
   - Run `go generate ./...`
```

Reply:
I meant to only re-generate this changed receiver

Reply:
Right, for some receivers (ie `hostmetricsreceiver`) the metadata file exists in subdirectories.